### PR TITLE
Favicon Added

### PR DESCRIPTION
**Pull request by richarddushime**
Hi @bebatut @NPDebs 
I thought of adding the Favicon as i had suggested in the #628  issue  
i added 2  icons in the images and one of them can be used depends on what you will prefer 
Hopefully it helps 
Screenshot 2023-06-27 160452

Thank you

### CODE REVIEW

**Comment by NPDebs** [comment]
Hi, @richarddushime.
Thank you for your work here! ✨
The favicon gives the website a nice feeling.

As mentioned in an earlier [blogpost](https://openlifesci.org/posts/2023/04/18/Introducing-the-new-OLS/) , we are working on rebranding the website this summer. As such, major design changes are being implemented in a calculated and organised pattern.

That said, this looks good to merge. 😻 Thank you!

**Comment by richarddushime** [comment]
> Hi, @richarddushime. Thank you for your work here! ✨ The favicon gives the website a nice feeling.
> 
> As mentioned in an earlier [blogpost](https://openlifesci.org/posts/2023/04/18/Introducing-the-new-OLS/) , we are working on rebranding the website this summer. As such, major design changes are being implemented in a calculated and organised pattern.
> 
> That said, this looks good to merge. 😻 Thank you!

Thanks , Thats Cool 
will rebranding be  Open Source ? i would be happy to contribute too

**Comment by NPDebs** [comment]
@richarddushime Yes, please!
The rebranding will be open source. Contributions are very welcome.

More details about this will be shared soon. Thank you, Richard.

**Comment by NPDebs** [comment]
Hey @richarddushime! 👋 

A little feedback on your PR:

- I like the size of this icon (`fav.ico`), but the white background doesn't sit well. 👇 
Screenshot of favicon with background

- This image `favicon.ico` looks much better, since the background has been removed. 👇 
Screenshot 2024-05-28 at 13 55 12

- However, it is **too small** and the logo is barely visible. I think the padding might be a bit much. 👇 
Screenshot of favicon.ico


**Short version:** `favicon.ico` is preferred, but with the padding removed. 
What do you think, @richarddushime? // @bebatut

**Comment by richarddushime** [comment]
the icon without the  white-bg looks great 
Thanks
